Library/Services/Implementation/NfieldSurveyInterviewSimulationService.cs

```diff
@@ -139,6 +166,16 @@ private Uri SurveySimulationHintsEndPoint(string surveyId)
             return new Uri(ConnectionClient.NfieldServerUri, $"surveys/{surveyId}/InterviewSimulations/DownloadHints");
         }
 
+        private Uri InterviewSimulationsEndPoint()
+        {
+            return new Uri(ConnectionClient.NfieldServerUri, "surveys/InterviewSimulations");
```

Comment:
Do we have a naming convention for the endpoints? Like camel case or something else? As `surveys` are starting with small letter but InterviewSimulations with a capital one. Not a problem per se, just for consistency in the code.

Reply:
I think this was a mistake. In WebApiConfig in public api all endpoints are defined with a capital. 
I don't think it is case sensitive, but I'll change it to capital.